terraform/modules/validators/stack_validators.tf

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2023, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+# The validation fails if any running/stopped compute instances with the display name "<service_name>-wls-0" are present in the stack compartment.
```

Comment:
Continue with the comment as to why we're doing this validation, "This is to prevent trying to create another instance using the same prefix, which will lead to DNS issues."

Reply:
Updated.